make_queue/templates/make_queue/course/registration_edit.html

```diff
@@ -32,6 +33,16 @@ <h1>
                 </div>
                 <p> {% trans "Are you sure you entered the correct username?" %} </p>
             </div>
+            {% form_has_error form "username" "unique" as unique_error %}
```

Comment:
I think we should use named arguments for the `field` and `code` arguments when calling the template tag. That would make the code more readable. 

```{% form_has_error form field="username" code="unique" as unique_error %}```